frontned/src/providers/theme.ts

```diff
@@ -1,7 +1,19 @@
 import { createTheme } from '@mui/material/styles';
 
 const theme = createTheme({
-  palette: {},
+  palette: {
+    primary: {
+      main: '#43A047',
+    },
+    text: {
+      primary: '#000000DE',
+      secondary: '#00000099',
+      disabled: '#00000061;',
+    },
```

Comment:
Remember that you can customise the components here as well for all the app (for example buttons might have some common attributes across the app.

Reply:
As for now, it is quite hard for me to pick common styles for certain components  - but I will keep that in mind, and in the case where I can use it - I definitely will